Mathlib/CategoryTheory/Enriched/Basic.lean

```diff
@@ -63,7 +63,7 @@ class EnrichedCategory (C : Type u₁) where
   assoc (W X Y Z : C) : (α_ _ _ _).inv ≫ comp W X Y ▷ _ ≫ comp W Y Z =
     _ ◁ comp X Y Z ≫ comp W X Z := by aesop_cat
 
-@[inherit_doc EnrichedCategory.Hom] notation X " ⟶[" V "] " Y:10 => (EnrichedCategory.Hom X Y : V)
+@[inherit_doc EnrichedCategory.Hom] notation3 X " ⟶[" V "] " Y:10 => (EnrichedCategory.Hom X Y : V)
```

Comment:
I don’t know the difference between notation and notation3 very well; what is the motivation behind this change?